cli_tools/gce_ovf_import/ovf_importer/ovf_importer.go

```diff
@@ -483,3 +515,41 @@ func (oi *OVFImporter) getOsIDValue(descriptor *ovf.Envelope) (osIDValue string,
 	}
 	return osIDValue, nil
 }
+
+func (oi *OVFImporter) buildBootDiskImageImportRequest(imageName string, bootDiskFilePath string) (request importer.ImageImportRequest, err error) {
+	bootDiskFilePathSource, err := importer.NewSourceFactory(oi.storageClient).Init(bootDiskFilePath, "")
+
+	if err != nil {
+		return request, err
+	}
+
+	request = importer.ImageImportRequest{
+		ExecutionID:           imageName,
```

Comment:
Do you know why this `ExecutionID` is not generated but just assigned an imageName? In other places it is a build ID, I am a bit confused here.

Reply:
The executionID is used as a substring of the disks names that are created in the inflation step, so I think it's make sense to be assigned an imageName, also it was name like this in the old [multiImage-importer](https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/cli_tools/gce_ovf_import/multi_image_importer/request_builder.go#L41)
but, really i don't know when it should be assigned as BuildID, @EricEdens could you please clarify this part for us?